compiler/lambdalifting.nim

```diff
@@ -530,9 +529,9 @@ type
     unownedEnvVars: Table[int, PNode] # only required for --newruntime
 
 proc initLiftingPass(fn: PSym): LiftingPass =
-  result.processed = initIntSet()
+  result = LiftingPass(processed: initIntSet(),
```

Comment:
Bikeshedding but `toIntSet [fn.id]` would move all the construction into the constructor, since that seems to be the point here. Same with the other case.

Reply:
Thanks!

Reply:
Though, the goal is to assign something to result because of the strict definition analysis. The old code should have given warnings, but there is a bug related to this. Anyway, moving everything into the construction looks more concise and elegant.